packages/driver/package.json

```diff
@@ -5,7 +5,9 @@
   "scripts": {
     "clean-deps": "rm -rf node_modules",
     "cypress:open": "node ../../scripts/cypress open",
-    "cypress:run": "node ../../scripts/cypress run",
+    "cypress:run": "node ../../scripts/cypress run --spec \"cypress/integration/*/*\",\"cypress/integration/*/!(multi-domain)/**/*\"",
+    "cypress:open-experimentalSessionAndOrigin": "node ../../scripts/cypress open --config experimentalSessionAndOrigin=true",
+    "cypress:run-experimentalSessionAndOrigin": "node ../../scripts/cypress run --config experimentalSessionAndOrigin=true  --spec \"cypress/integration/e2e/multi-domain/**/*\"",
```

Comment:
Would it make sense to test all files with the flag on to ensure no one adds anything that will break once we turn on the flag?

Reply:
The tests that changed in the navigation spec will fail if they're run. I could look at flexing them base don the experimental flag maybe?